sdks/auth_aad/Cargo.toml

```diff
@@ -29,8 +29,8 @@ async-trait          = "0.1.36"
 
 [dev-dependencies]
 tokio                  = { version = "0.2", features = ["macros"] }
-azure_storage          = { version = "0.1", path = "../storage" }
+azure_storage          = { version = "0.2", path = "../storage" }
```

Comment:
I wouldn't bump this. We haven't even shipped 0.1 yet so we'd essentially be skipping a 0.1 release.

Reply:
Makes sense, I'll revert the version.